Sources/MUXSDKStatsKaltura/MUXSDKPlayerBinding.swift

```diff
@@ -6,16 +6,18 @@
 //  Copyright © 2021 Mux, Inc. All rights reserved.
 //
 
+import AVFoundation
 import Foundation
-import PlayKit
+import UIKit
+
 import MuxCore
-import AVFoundation
+import PlayKit
 
 @objc
 public class MUXSDKPlayerBinding: NSObject {
     private let MUXSDKPluginName = "apple-kaltura-mux"
-    private let MUXSDKPluginVersion = Bundle.main.infoDictionary?["CFBundleVersion"] as? String
-    
+    private let MUXSDKPluginVersion = "3.0.0"
```

Comment:
not a big deal, but just curious why this is preferable

Reply:
Great question! 

`Bundle.main.infoDictionary?["CFBundleVersion"]` no longer returns the SDK version with certainty. 

When installing the SDK as a Swift Package, this API will return the application `CFBundleVersion` instead. With Cocoapods it's a bit more complicated but there's not guarantee `CFBundleVersion` matches the pod spec version. Instead it depends on the structure that Cocoapods will setup, which varies based on the spec, the Podfile, and the Cocoapods version.

Reply:
This is because there's no SDK binary and by extension no `Info.plist` file.

Reply:
Thanks for the in-depth answer! 🥇

Reply:
I do have some automation in mind for verifying this constant is what it should be, similar to how we verify the pod spec version using a script.